doc/_extensions/odoo_ext/layout.html

```diff
@@ -26,14 +26,15 @@
 
 {%- block footer -%}
   {%- if google_analytics_key -%}
+    <script id="tracking_code" async src="https://www.googletagmanager.com/gtag/js?id={{ google_analytics_key }}"></script>
     <script>
-    (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
-    (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
-    m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
-    })(window,document,'script','//www.google-analytics.com/analytics.js','ga');
+      window.dataLayer = window.dataLayer || [];
+      function gtag(){dataLayer.push(arguments);}
+      gtag('js', new Date());
 
-    ga('create', '{{ google_analytics_key }}', 'auto');
-    ga('send','pageview');
+      gtag('config', '{{ google_analytics_key }}', {
+        'anonymize_ip': {{ google_analytics_anonymize and 'true' or 'false' }},
```

Comment:
I dont see it in the config of the doc ;) As explained last week (jinja/qweb), this page is special, something from the documentation build with shinx ;)
So you can just remove the anonymize_ip here in this file, it is not linked...